doc/DeveloperGuide.md

```diff
@@ -88,6 +88,22 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Rename existing tag
+
+**MSS**
+
+1. User requests to rename an existing tag
+2. AddressBook shows a list of persons with the existing tag
+3. AddressBook confirms with user about his/her decision to rename the existing tag
```

Comment:
The use case seeks to model the interaction between an actor and the system.
Therefore the interaction should be alternating between user and the system, `user -> system -> user -> system -> ...`
In this MSS, do you think there's something that could be changed to reflect this?